### PR TITLE
Cloudinary-service_CU-mt99tf

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-mt99tf Cloudinary service](https://app.clickup.com/t/mt99tf)

**Comment by lpboucher** [comment]
Task linked: [CU-mt99tf Cloudinary service](https://app.clickup.com/t/mt99tf)

File: src/app/core/views/image.service.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { environment as config } from "@env";
+
+@Injectable({
+    providedIn: "root"
+})
+export class ImageService {
+
+    constructor() { }
+
+    getImageUrl(path: string): string {
```
Comment (lpboucher) [inline src/app/core/views/image.service.ts]: can be private

File: src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const environment = {
   production: true,
-  backendURL: "https://dakcoffeeroasters.com/api/"
+  backendURL: "https://dakcoffeeroasters.com/api/",
+  cloudinaryName: { cloudName: "dak-coffee-roasters" },
```
Comment (lpboucher) [inline src/environments/environment.prod.ts]: should be a string instead of an object

Comment (lpboucher) [reply src/environments/environment.prod.ts]: done

File: src/app/core/views/image.service.ts (hunk 1 above)
Comment (lpboucher) [reply src/app/core/views/image.service.ts]: done

**Review by lpboucher — APPROVED** [review]

**Pull request merged**